packages/components/src/components/date-picker/date-picker-input.jsx

```diff
@@ -37,11 +37,21 @@ const DatePickerInput = ({
         setIsClearBtnVisible(false);
     };
 
+    const OpenIcon = (
+        <DatePickerIcon
+            onClick={onClick}
+            className={classNames({
+                'dc-datepicker__icon': !!onClick, // If has an onclick function add class
```

Comment:
```suggestion
                'dc-datepicker__icon': typeof onClick === 'function', // If has an onclick function add class
```

Reply:
Thank you Salar for your review.
We already checked type of the "onClick" prop in PropTypes. Here we only need to know if it is not "null".
I didn't use Boolean function because !! is shorter code to check if some value is True of False.